plugins/polio/admin.py

```diff
@@ -170,6 +171,35 @@ class VaccineRequestFormAdmin(admin.ModelAdmin):
     readonly_fields = ["created_at", "updated_at"]
     list_display = ["campaign", "get_country", "count_pre_alerts", "count_arrival_reports", "created_at"]
 
+    def save_related(self, request, form, formsets, change):
+        for formset in formsets:
+            if not formset.is_valid():
+                print(f"Formset errors: {formset.errors}")
```

Comment:
Shouldn't we do something more useful than a `print()` here?(e.g. showing an error to the use or logging the error)